Extension/tslint.json

```diff
@@ -1,31 +1,184 @@
 {
     "rules": {
+        "adjacent-overload-signatures": true,
+        "align": true,
+        "array-type": false,
+        "arrow-parens": false,
+        "arrow-return-shorthand": true,
+        "await-promise": false,
+        "ban-comma-operator": true,
+        "binary-expression-operand-order": true,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": true,
+        "completed-docs": false,
+        "no-console": false,
```

Comment:
I'm pro `no-console` since the extension should be writing to its own output window and not the developer console for electron. 
https://palantir.github.io/tslint/rules/no-console/